### PR TITLE
[NCL-7221] Implement Product List page

### PR DESCRIPTION
**Pull request by patrikk0123**

### CODE REVIEW

File: src/components/ProductsPage/ProductsPage.tsx [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
-
-      <PageSection>
-        <ContentBox padding>
-          <ActionButton link="/products/100/versions/100/milestones/101/details">
-            Product Milestone (testing purposes)
-          </ActionButton>
-        </ContentBox>
-      </PageSection>
-    </>
+    <PageLayout title={PageTitles.products} description={<>This page contains products... TODO</>}>
```
Comment (matedo1) [inline src/components/ProductsPage/ProductsPage.tsx:26]: Description `This page contains list of products like <Label>EAP</Label>, usually product is a deliverable package composed of multiple projects like <Label>JBoss Modules</Label> or <Label>Hibernate<Label>.`

Comment (matedo1) [reply src/components/ProductsPage/ProductsPage.tsx:26]: @jbartece is there anything else you would like to propose for the page description that will be displayed on product list page?

File: src/components/ProductsPage/ProductsPage.tsx [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
+      <ContentBox shadow={false} background={false} marginBottom>
```
Comment (matedo1) [inline src/components/ProductsPage/ProductsPage.tsx:27]: Why `ContentBox` is needed?

**Review by matedo1 — CHANGES_REQUESTED** [review]

File: src/components/ProductsList/ProductsList.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      placeholder: 'string | !string | s?ring | st*ng',
+      operator: '=like=',
+    },
+  },
+};
+
+const sortOptions: ISortOptions = {
+  name: {
+    id: 'name',
+    title: 'Name',
```
[the remark this answers is not in the document]
Comment (patrikk0123) [reply src/components/ProductsList/ProductsList.tsx:39]: Not for the time being, sorting options structure needs to be changed first. Then title can be removed here and also on **ProjectsList** component.

**Review by matedo1 — APPROVED** [review]

**Review by matedo1 — APPROVED** [review]

**Pull request merged**